### PR TITLE
[DO NOT MERGE] upgrade-nebula

### PR DESCRIPTION
**Pull request by cooper-lzy**

### CODE REVIEW

File: docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    $ sudo /usr/local/nebula_GA/scripts/nebula.service start metad
+    ```
+
+5. 检查Meta服务是否正常。
+
+    ```bash
+    $ sudo /usr/local/nebula_GA/scripts/nebula.service status metad
+    ```
+
+    如果服务异常，请查看目录`/usr/local/nebula_GA/logs`内的meta相关日志排查故障。
```
Comment (tom-chensf) [inline docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph.md:83]: add meta log file ，give some example about failed message

**Review by whitewum — CHANGES_REQUESTED** [review]
Wait a sec. 
Some steps are error-prone.

File: docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+## 升级步骤
+
+1. 安装新版本Nebula Graph，保证配置和旧版本一致。
+
+    >**注意**：外围服务都需要修改为新版本端口，例如Nebula Exchange、Nebula Importer、Nebula Console等。
+
+2. 停止旧版本的Nebula Graph服务。
+
+    ```bash
+    $ sudo usr/local/nebula/scripts/nebula.service stop all
```
Comment (critical27) [inline docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph.md:58]: /usr/local/nebula/scripts/nebula.service

File: docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    ```
+
+3. 将旧版本的目录`data/meta`复制到新版本的相同位置。
+
+    >**说明**：目录`data/meta`由配置文件`nebula-metad.conf`中的参数`--data_path`指定。
+
+    ```bash
+    $ sudo cp -R /usr/local/nebula/data/meta/ /usr/local/nebula_GA/data/meta/
```
Comment (critical27) [inline docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph.md:66]: add some description about where `data/meta` is from.

File: docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+    --dst_db_path=<new_storage_directory_path>  \
+    --upgrade_meta_server=<meta_server_ip1>:<meta_server_port1>[,<meta_server_ip2>:<meta_server_port2>..] \
+    --upgrade_version=<old_nebula_version> \
+    ```
+
+    示例如下：
+
+    ```bash
+    $ sudo /usr/local/nebula_GA/bin/db_upgrader \
+    --src_db_path=/usr/local/nebula/data/storage/ \
```
Comment (critical27) [inline docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph.md:104]: Add some description about where data/storage is from. Add a example when data_path consists of multiple path if necessary.

File: docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+    --dst_db_path=/usr/local/nebula_GA/data/storage/ \
+    --upgrade_meta_server=127.0.0.1:9559 \
+    --upgrade_version=1
```
Comment (critical27) [inline docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph.md:107]: `upgrade_version` ditto

File: docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+    ```
+
+7. 启动新版本的Storage服务，并检查服务是否正常。
```
Comment (critical27) [inline docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph.md:110]: 加句等所有机器升级工具都运行完之类的？

File: docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph.md [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+
+5. 检查Meta服务是否正常。
+
+    ```bash
+    $ sudo /usr/local/nebula_GA/scripts/nebula.service status metad
+    ```
+
+    如果服务异常，请查看目录`/usr/local/nebula_GA/logs`内的meta相关日志排查故障。
+
+6. 执行如下命令升级版本。集群部署时，需要在集群中的所有Storage服务节点都执行此命令。
```
Comment (critical27) [inline docs-2.0/4.deployment-and-installation/3.upgrade-nebula-graph.md:88]: **所有Storage**加粗之类的强调下

**Pull request closed without merging**